unicodetools/data/ucd/dev/UnicodeData.txt

```diff
@@ -19327,6 +19327,75 @@ FFFD;REPLACEMENT CHARACTER;So;0;ON;;;;;N;;;;;
 10D37;HANIFI ROHINGYA DIGIT SEVEN;Nd;0;AN;;7;7;7;N;;;;;
 10D38;HANIFI ROHINGYA DIGIT EIGHT;Nd;0;AN;;8;8;8;N;;;;;
 10D39;HANIFI ROHINGYA DIGIT NINE;Nd;0;AN;;9;9;9;N;;;;;
+10D40;GARAY DIGIT ZERO;Nd;0;AN;;0;0;0;N;;;;;
+10D41;GARAY DIGIT ONE;Nd;0;AN;;1;1;1;N;;;;;
+10D42;GARAY DIGIT TWO;Nd;0;AN;;2;2;2;N;;;;;
+10D43;GARAY DIGIT THREE;Nd;0;AN;;3;3;3;N;;;;;
+10D44;GARAY DIGIT FOUR;Nd;0;AN;;4;4;4;N;;;;;
+10D45;GARAY DIGIT FIVE;Nd;0;AN;;5;5;5;N;;;;;
+10D46;GARAY DIGIT SIX;Nd;0;AN;;6;6;6;N;;;;;
+10D47;GARAY DIGIT SEVEN;Nd;0;AN;;7;7;7;N;;;;;
+10D48;GARAY DIGIT EIGHT;Nd;0;AN;;8;8;8;N;;;;;
+10D49;GARAY DIGIT NINE;Nd;0;AN;;9;9;9;N;;;;;
+10D4A;GARAY VOWEL SIGN A;Lo;0;R;;;;;N;;;;;
+10D4B;GARAY VOWEL SIGN I;Lo;0;R;;;;;N;;;;;
+10D4C;GARAY VOWEL SIGN O;Lo;0;R;;;;;N;;;;;
+10D4D;GARAY VOWEL SIGN EE;Lo;0;R;;;;;N;;;;;
```

Comment:
FYI The proposal calls this "vowel sign EU" as well as "vowel sign EE": https://github.com/unicode-org/sah/issues/55#issuecomment-1766934556